### DISCUSSION TITLE
Welcome to Seeneva Discussions!

### DISCUSSION THREAD

**Discussion by Zellius** [Seeneva/seeneva-reader-android#15] [General]
# 👋 Welcome!

Thank you for your interest in the **Seeneva**. It is only a beginning of the project and in my plans to make it smarter and better :smiley: So all comic book fans will get a new open source way how to read comics. If you have any question, you can ask it here.

**Comment by sid350** [comment]
How to switch direction to right-left to read manga?

**Reply by a-t-h-a-r-v** [reply]
![IMG_20240914_231858.jpg](https://github.com/user-attachments/assets/805bda7a-66b1-4676-8b6b-9459c54d85c7)
